python/run_stubtest.py

```diff
@@ -0,0 +1,14 @@
+import os
+import sys
+import tempfile
+
+from mypy import stubtest
+
+if __name__ == "__main__":
+    file_descriptor, file_name = tempfile.mkstemp(text=True)
```

Comment:
This seems quite low-level &ndash; would something like [`NamdTemporaryFile`](https://docs.python.org/2.7/library/tempfile.html#tempfile.NamedTemporaryFile) make things easier?

Reply:
done -- unfortunately on Windows such a file can't be opened multiple times, which would remove another two lines.